.gitmodules

```diff
@@ -216,3 +216,6 @@
 [submodule "assets/syntaxes/02_Extra/gnuplot"]
 	path = assets/syntaxes/02_Extra/gnuplot
 	url = https://github.com/hesstobi/sublime_gnuplot
+[submodule "assets/syntaxes/02_Extra/SystemVerilog"]
+	path = assets/syntaxes/02_Extra/SystemVerilog
+	url = git@github.com:TheClams/SystemVerilog.git
```

Comment:
All other submodules have a `https` link. Not sure if that could cause any problems, but might make sense to change it even if only for the sake of consistency.

Reply:
I think it should be changed away from SSH, yes.